src/YieldDisburser.sol

```diff
@@ -2,16 +2,16 @@
 pragma solidity ^0.8.22;
 
 import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
-import {Checkpoints} from "openzeppelin-contracts/contracts/utils/structs/Checkpoints.sol";
+import {Checkpoints} from "openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/utils/structs/Checkpoints.sol";
 import {Time} from "openzeppelin-contracts/contracts/utils/types/Time.sol";
-import {IBreadToken} from "./IBreadToken.sol";
+import {Bread} from "../lib/bread-token-v2/src/Bread.sol";
 
 error AlreadyClaimed();
 
 contract YieldDisburser is OwnableUpgradeable {
     address[] public breadchainProjects;
     address[] public breadchainVoters;
-    IBreadToken public breadToken;
+    Bread public breadToken;
```

Comment:
I think this should still be an interface contract that we're importing. Otherwise it will include the code of the entire contract, which we dont need

Reply:
it's only going to increase the deployment cost by very little because it's gnosis, but I can open a pr for an interface on the original repo to add an interface